.golangci.yaml

```diff
@@ -82,7 +82,7 @@ linters:
     - deadcode
     - revive
     - exportloopref
-    - godox
+    # - godox # Disabled as a TODO should not block code from passing
```

Comment:
Justification for me to add that was that TODO should be transformed to issue at the time of the PR merge and code should not hold any task tracking elements as that is not the main reason. 

Ok to keep it. Just giving context

Reply:
Understood. I think the difficult here is that the TODOs are for code which do not currently exist on the main branch. I will remove the TODOs though as what you say makes sense.